Firestore/Example/Tests/Local/FSTLevelDBQueryCacheTests.mm

```diff
@@ -91,7 +91,7 @@ - (void)testMetadataPersistedAcrossRestarts {
   SnapshotVersion lastVersion(Timestamp(1, 2));
 
   db1.run("add query data", [&]() {
-    FSTQuery *query = [FSTQuery queryWithPath:ResourcePath{"some", "path"}];
+    core::Query query = Query("some/path");
```

Comment:
You could std::move query on the next line. (Or just inline it.)

Reply:
Done.